scripts/core/editor3/controllers/Editor3Controller.jsx

```diff
@@ -11,14 +11,24 @@ import {Editor3} from '../components/Editor3';
  * @see sdEditor3
  */
 
-Editor3Controller.$inject = ['$element'];
-export function Editor3Controller($element) {
-    var showToolbar = true;
+Editor3Controller.$inject = ['$element', 'spellcheck'];
+export function Editor3Controller($element, spellcheck) {
+    let showToolbar = true;
 
     if (!this.editorFormat || this.readOnly) {
         showToolbar = false;
     }
 
+    let spellchecker = new DraftSpellchecker('en');
```

Comment:
In my opinion, it doesn't make sense to initialise the spellchecker here, outside the editor. It is functionality that belongs to the editor, so it should be initialised inside. If we ever decide that we want to disable it, we can always do so via editor options.

Instead, I would simply pass the `spellcheck` service as a prop to `Editor3`, so that it can take advantage of the whole API.

It would make more sense and would be cleaner...